src/chat/components/CustomImageVideoBubble.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useRef, useState } from 'react';
+import {
+  StyleSheet,
+  View,
+  Pressable,
+  Image,
+  TouchableOpacity,
+} from 'react-native';
+import Video, { VideoRef } from 'react-native-video';
+import FastImage from 'react-native-fast-image';
+import { MessageTypes, type MessageProps } from '../../interfaces';
+
+interface CustomImageVideoBubbleProps {
+  message: MessageProps;
+  position: 'left' | 'right';
+  selectedImgVideoUrl: (url: string) => void;
+}
+
+export const CustomImageVideoBubble: React.FC<CustomImageVideoBubbleProps> = ({
+  position,
+  message,
+  selectedImgVideoUrl,
+}) => {
+  const [isPauseVideo, setIsPauseVideo] = useState(true);
+  const videoRefs = useRef<VideoRef>(null);
+
+  return (
+    <View
+      style={[
+        styles.bubleContainer,
+        position === 'left' ? styles.flexStart : styles.flexEnd,
+      ]}
+    >
+      <Pressable
+        onPress={() =>
+          message.path &&
+          message.type === MessageTypes.image &&
+          selectedImgVideoUrl(message.path)
+        }
+        style={styles.bubble}
+      >
+        {message.type === MessageTypes.image ? (
```

Comment:
please create a function for each type and add props to custom style and the render of each type

Reply:
ok. I already separate to small function and add custom style for it